mne_bids/utils.py

```diff
@@ -26,6 +26,15 @@
 from .tsv_handler import _to_tsv, _tsv_to_str
 
 
+def _get_ch_type_mapping():
```

Comment:
Do you really need a function for this :-) http://matthewrocklin.com/blog/work/2019/06/23/avoid-indirection

Reply:
We had some code very similar to this in `write.py` written by @monkeyman192 ... I am refactoring this a bit to avoid code duplication. You can see it in this commit: https://github.com/mne-tools/mne-bids/pull/219/commits/36f74031709f173a195813a44af316953f9071d5